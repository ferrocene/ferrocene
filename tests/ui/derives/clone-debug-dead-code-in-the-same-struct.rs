#![forbid(dead_code)]

#[derive(Debug)]
pub struct Whatever { //~ ERROR struct `Whatever` is never constructed
    pub field0: (),
    field1: (),
    field2: (),
    field3: (),
    field4: (),
}

fn main() {}
