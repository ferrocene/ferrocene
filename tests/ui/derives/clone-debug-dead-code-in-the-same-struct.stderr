error: struct `Whatever` is never constructed
  --> $DIR/clone-debug-dead-code-in-the-same-struct.rs:4:12
   |
LL | pub struct Whatever {
   |            ^^^^^^^^
   |
note: the lint level is defined here
  --> $DIR/clone-debug-dead-code-in-the-same-struct.rs:1:11
   |
LL | #![forbid(dead_code)]
   |           ^^^^^^^^^

error: aborting due to 1 previous error

